### ISSUE TITLE
The Initial_learning_rate just can recongnition '1'?

### ISSUE THREAD

**Issue by lmolhw5252** [emedvedev/attention-ocr#74]
When I want to change learning rate, I found that the learning rate cannot recognition bellow '1',
![image](https://user-images.githubusercontent.com/20182607/38164781-1c544c92-353c-11e8-89e4-452df23e9ae5.png)

**Comment by emedvedev** [comment]
@lmolhw5252 it's working with values <1 as it should, just the display value was casted to int. Fixed now.
